### PR TITLE
Use Granite SeekBar

### PR DESCRIPTION
**Pull request by markodp**

### CODE REVIEW

File: src/Widgets/Player/TimeWidget.vala [hunk 1]
```diff
@@ -19,30 +19,20 @@
...
+public class Audience.Widgets.TimeWidget : Granite.SeekBar {
     unowned ClutterGst.Playback main_playback;
-    public Gtk.Label progression_label;
-    public Gtk.Label time_label;
-    public Gtk.Scale scale;
     private Audience.Widgets.PreviewPopover preview_popover;
-    private bool released = true;
 
     public TimeWidget (ClutterGst.Playback main_playback) {
+        base(0.0);
```
Comment (Philip-Scott) [inline src/Widgets/Player/TimeWidget.vala]: Use GObject style construction :) 

`Object (playback_duration: 0.0);`

**Comment by markodp** [comment]
no problem, done :)

**Comment by Philip-Scott** [comment]
waiting for Cody's stable release

**Comment by codygarver** [comment]
Build dependencies should require the minimum Granite version that has SeekBar

**Pull request merged**